network-store-iidm-impl/src/test/java/com/powsybl/network/store/iidm/impl/LineTest.java

```diff
@@ -35,7 +35,6 @@
 public class LineTest {
 
     //TODO: there is a similar test in the TCK tests. A CurrentLimitsTest extends AbstractCurrentLimitsTest should be created and this test can be deleted.
-    // The TCK test doesn't pass yet. As is, the network-store implementation of setV(v) on buses is not consistent. We have problems with the views we are working on (BusBreakerView or BusView).
```

Comment:
We should delete this test now as suggested by the comment since we have the CurrentLimitsTest in our TCK that passes

Reply:
Done